spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -259,6 +262,42 @@ public PersistentStreamMessageSourceRegistrar persistentStreamRegistrar(
         return new PersistentStreamMessageSourceRegistrar(environment, executorBuilder);
     }
 
+    /**
+     * Creates a {@link ConfigurerModule} to set
+     * {@link EventProcessingConfigurer.SubscribableMessageSourceDefinitionBuilder}
+     *
+     * @param executorBuilder         The {@link java.util.concurrent.ScheduledExecutorService} builder used during
+     *                                construction of the {@link PersistentStreamMessageSourceDefinition}.
+     * @param psFactory               used during construction of the {@link PersistentStreamMessageSourceDefinition}.
+     * @param axonServerConfiguration Contains the persistent stream settings.
+     * @return A {@link ConfigurerModule} to configure
+     */
+    @Bean
+    @ConditionalOnProperty(name = "axon.axonserver.auto-persistent-streams-enable")
+    public ConfigurerModule autoPersistentStreamMessageSourceDefinitionBuilder(
+            PersistentStreamScheduledExecutorBuilder executorBuilder,
+            PersistentStreamMessageSourceFactory psFactory,
+            AxonServerConfiguration axonServerConfiguration) {
+        AxonServerConfiguration.PersistentStreamSettings psSettings = axonServerConfiguration.getAutoPersistentStreamsSettings();
+        return configurer -> configurer.eventProcessing().usingSubscribingEventProcessors(
+                processingGroupName -> {
+                    String psName = processingGroupName + "-stream";
```

Comment:
I think it would be good to document this default naming choice. My gut tells me the `AxonServerConfiguration` properties you've added are the right spot for this in the JavaDoc. Additionally, having the default naming in the `subscribing.adoc` documentation wouldn't hurt either.